src/engine/qcommon/parse.cpp

```diff
@@ -2377,30 +2379,31 @@ static int Parse_EvaluateTokens( source_t *source, token_t *tokens, signed long
 									break;
 								}
 							}
+							DAEMON_FALLTHROUGH;
 
-						case punctuationSub_t::P_MUL:
-						case punctuationSub_t::P_DIV:
-						case punctuationSub_t::P_MOD:
-						case punctuationSub_t::P_ADD:
+						case punctuationSub_t::P_MUL: DAEMON_FALLTHROUGH;
```

Comment:
It seems that here and in other places in the CL, having multiple case statements together shouldn't require FALLTHROUGH.

Reply:
Hm, maybe. I was over zealous. I will remove some and see if the warnings come back.